cmd/snap/cmd_connect_test.go

```diff
@@ -62,24 +62,31 @@ Help Options:
 
 func (s *SnapSuite) TestConnectExplicitEverything(c *C) {
 	s.RedirectClientToTestServer(func(w http.ResponseWriter, r *http.Request) {
```

Comment:
Ah, quite interesting. So we POST, get an change ID (``zzz``) and then we poll that change by GETting a 2nd URL, right?